api.yaml

```diff
@@ -220,12 +220,146 @@ paths:
             $ref: '#/components/responses/ApiKeyInsufficientPermissions'
           '429':
             $ref: '#/components/responses/TooManyRequests'
+            
+    /search/tests:
```

Comment:
why is this `/search/tests` instead of `/tests/search`? this is not really restfull - wich usually I don't really care about but in public api we should maybe care more